templates/blog/macros.html

```diff
@@ -21,7 +21,7 @@
     <a href={{ url }} lang={{ article.language }}>
       <h1>{{ article.title|title }}</h1>
       <figure>
-        <img src="{{ article.image | default('/static/images/articles/default.png', true) }}" width=100%>
+        <img src="{{ article.image | default('/static/images/article/default.png', true) }}" width=100%>
```

Comment:
bewarn that the _article**s**_ (with an "s") folder must exist and is used for uploaded contents.
It may be a better place to keep it in the same folder as uploaded.
The difference would be that _articles/default.png_ would be taken by default and _articles/1234/5678_ would be taken for image 5678 of article 1234.